spec/models/form526_submission_spec.rb

```diff
@@ -203,10 +214,47 @@
 
     describe 'accepted_to_primary_path' do
       it 'returns submissions with a submitted_claim_id' do
+        expect(Form526Submission.accepted_to_evss_primary_path).to contain_exactly(
```

Comment:
Wow nice clean tests I like the way you set this up, and covered these use cases in order!